src/ShapeCrawler/Tables/ITable.cs

```diff
@@ -39,6 +39,11 @@ public interface ITable : IShape
     ///     Merge neighbor cells.
     /// </summary>
     void MergeCells(ICell cell1, ICell cell2);
+
+    /// <summary>
+    ///     Deletes a column at specified index
```

Comment:
The build fails here with "Documentation text should end with a period". Please build the **Release** configuration on your local machine to see all code style issues.